src/controllers/verifyEmail.js

```diff
@@ -0,0 +1,15 @@
+import { User } from '../sequelize/models/index';
```

Comment:
This should be part of the userController

Reply:
Excuse me... You mean the whole code inside the verifyEmail.js file, should be inside the userController.js file? Or, that single line of importing from sequelize should be inside the userController?

Reply:
I mean the whole code inside the verifyEmail.js file

Reply:
Okay.